sub/sd_ass.c

```diff
@@ -23,6 +23,7 @@
 
 #include <libavutil/common.h>
 #include <ass/ass.h>
+#include <ass/ass_types.h>
```

Comment:
I just checked, and ass.h indeed already includes ass_types.h. Remove it and I can merge this.